packages/app/tsconfig.eslint.json

```diff
@@ -0,0 +1,4 @@
+{
+  "extends": "./tsconfig.json",
+  "include": ["src/**/*.ts", "src/**/*.js", ".eslintrc.js"]
+}
```

Comment:
Linter Error ; how important is this with the new lint settings?

Reply:
It changes the files which will and can be checked by eslint.